src/crimson/os/seastore/omap_manager/btree/omap_btree_node.h

```diff
@@ -15,6 +15,9 @@
 
 namespace crimson::os::seastore::omap_manager{
 
+const std::string BEGIN_KEY = "";
+const std::string END_KEY(64, (char)(-1));
```

Comment:
Not sure whether this is a reasonable max key, can we assert in the insert method that any key inserted must be less than this `END_KEY` ?